app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api::V1
+  class UsersController < ApiController
+
+    def show
```

Comment:
@yunixon move it to action `current`. I'm pretty sure we will need `show` action in future to get user by id.